cli/cmd/root.go

```diff
@@ -0,0 +1,37 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+	"html/template"
+	"os"
+)
+
+type Metadata struct {
+	MainClassName string
+	ArtifactName string
+}
+
+var rootCmd = &cobra.Command{
+	Use:   "jiny-cli",
```

Comment:
the '-' might not be necessary, I personally hate typing '-' because it's far and uses pinky (weak finger)

Reply:
As you wish 😏